crates/sui-core/src/checkpoints/mod.rs

```diff
@@ -754,10 +760,8 @@ impl CheckpointBuilder {
 
         let mut checkpoints = Vec::with_capacity(chunks_count);
         debug!(
-            "Creating {} checkpoints with {} transactions total after sequence {:?}",
-            chunks_count,
-            total,
-            last_checkpoint.as_ref().map(|(seq, _)| *seq)
+            ?last_checkpoint_seq,
```

Comment:
can we just call it `checkpoint_seq`?

Reply:
No, because this is the last checkpoint seq prior to the checkpoints to be constructed.